packages/search-in-workspace/src/browser/search-in-workspace-widget.tsx

```diff
@@ -100,6 +100,8 @@ export class SearchInWorkspaceWidget extends BaseWidget implements StatefulWidge
     @inject(SearchInWorkspacePreferences)
     protected readonly searchInWorkspacePreferences: SearchInWorkspacePreferences;
 
+    protected searchFormContainerRoot: Root;
```

Comment:
Minor: Again, what about making this readonly?

Reply:
It does not work here, because the `searchFormContainerRoot` is created in the post construct callback. I guess the code could be moved to the constructor, the class does not currently have one. Would you prefer that?

Reply:
It doesn't really bother me too much, it's fine 👍